### DISCUSSION TITLE
Future of AtlasMap

### DISCUSSION THREAD

**Discussion by mriekki** [atlasmap/atlasmap#4605] [General]
Can anyone comment on the future of AtlasMap.  It seems the amount of activity has gone down significantly, and it's been a year since the last release.  

Thanks
Mark

**Comment by arockiaraj1994** [comment]
Hi @mriekki, 

I had a same thought over the year, I think we have jump into the project and start contributing

**Reply by mattrpav** [reply]
Anyone have JS/front end experience?

I'd like the first release to be a simple dependency upgrade and refresh to stabilize things before looking at features, etc.

**Reply by perfetimperfect24** [reply]
Hi @mattrpav , I have Js/frontend experience and would love to contribute to atlasmap.

**Comment by mattrpav** [comment]
> Can anyone comment on the future of AtlasMap. It seems the amount of activity has gone down significantly, and it's been a year since the last release.
> 
> Thanks Mark

Hi Mark-

I am the original creator of AtlasMap and have been re-granted administrator permissions to the repo. Working to lay the groundwork to stabilize things.

Thanks,
Matt

**Reply by arockiaraj1994** [reply]
Hi Matt 

Thank you for the reply.

**Comment by arockiaraj1994** [comment]
Do we have any communication medium like zulip, slack or discord?

**Reply by mattrpav** [reply]
I created a Slack #atlasmap for now on the ASF Slack server -- https://the-asf.slack.com/archives/C083XG3F7JA

Request access using this discussion thread: https://github.com/atlasmap/atlasmap/discussions/4626
